index.html

```diff
@@ -1,20 +1,221 @@
+        <section class="container">
+          <article class="row">
+            <img
+              src="https://images.unsplash.com/photo-1576797723713-d11ed979e024?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1949&q=80"
+              class="about-me-photo sm-col-8 md-col-6 lg-col-6 col-12"
+            />
+            <img
+              src="https://images.unsplash.com/photo-1469580816584-a087c7601272?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1350&q=80"
+              class="about-me-photo sm-col-4 md-col-6 lg-col-6 col-12"
+            />
+          </article>
+        </section>
+      </section>
+      <!--More Sections to be added -->
+      <section class="black-and-white-image"></section>
+      <section class="about">
+        <div class="contact">
+          <h1 class="title">about</h1>
+          <div class="litte-intro-about">
+            <p>
+              Lorem Ipsum is simply dummy text of the printing and typesetting
+              industry. Lorem Ipsum has been the industry's standard dummy text
+              ever since the 1500s.Lorem Ipsum is simply dummy text of the
+              printing and typesetting industry.
+            </p>
+          </div>
+        </div>
+      </section>
+      <section class="air-ballon-image"></section>
+
+      <section class="container">
+        <div class="contact">
+          <h1 class="title">contact</h1>
+
+          <label for="fname" class="md-col-6 sm-col-10 star col-12"
+            >First Name</label
+          >
+          <input
+            type="text"
+            id="fname"
+            name="firstname"
+            placeholder="Your name.."
+            class="md-col-6 sm-col-10 box-height col-12"
+          />
+
+          <label for="lname" class="md-col-6 sm-col-10 star col-12"
+            >Last Name</label
+          >
+          <input
+            type="text"
+            id="lname"
+            name="lastname"
+            placeholder="Your last name.."
+            class="md-col-6 sm-col-10 box-height col-12"
+          />
+
+          <label for="email" class="md-col-6 sm-col-10 star col-12"
+            >Email</label
+          >
+          <input
+            type="text"
+            id="email"
+            name="email"
+            placeholder="Your email.."
+            class="md-col-6 sm-col-10 box-height col-12"
+          />
+
+          <label for="phone" class="md-col-6 sm-col-10 star col-12"
+            >Phone</label
+          >
+          <input
+            type="text"
+            id="phone"
+            name="phone"
+            placeholder="Your phone number.."
+            class="md-col-6 sm-col-10 box-height col-12"
+          />
 
-</body>
+          <label for="subject" class="md-col-6 sm-col-10 star col-12"
+            >Subject</label
+          >
+          <textarea
+            id="subject"
+            name="subject"
+            placeholder="Write something.."
+            style="height:200px"
```

Comment:
Avoid the `style` attribute. Instead, use a class like you do on other elements.